{{cookiecutter.project_slug}}/Makefile

```diff
@@ -85,7 +85,7 @@ NEURO?=neuro
 
 
 ifeq (${TRAIN_STREAM_LOGS}, yes)
-	_TRAIN_WAIT_START_OPTION=--wait-start
+	_TRAIN_WAIT_START_OPTION=--wait-start --detach
```

Comment:
Given this change we do not need to keep detach option here. You always want to detach your train job. 

Also, why is there an underscore before the name? Looks like we do not use such a naming scheme anywhere in Makefile (e.g. we use OPTION_GCP_CREDENTIALS). Could you please rename it to TRAIN_WAIT_START?

Reply:
thanks!